app/components/Configuration.react.js

```diff
@@ -21,10 +22,19 @@ class Configuration extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            isMenuOpen: false
+            isMenuOpen: false,
+            username: cookie.load('db-connector-user')
         }
         this.toggle = this.toggle.bind(this);
         this.close = this.close.bind(this);
+        this.logOut = this.logOut.bind(this);
+
+        if(window.require) {
+            window.require('electron').ipcRenderer.once('username',
+                (event, message) => {
+                    this.setState({username: message});
+                });
```

Comment:
could we add some comments here explaining what this is doing? Something along the lines of:
> If this is an electron app, then we don't have access to cookies and we don't require authentication in the API.
> In the browser setting, the username is set with a cookie but in electron this is set using electron's ipcRenderer.